WBNoticeViewSlidingMode introduced + bug fixes

troelsrichter opened the pull request:
Hello and thank you for making NoticeView

I've added the ability to slide the notice from the bottom up without braking the existing API. I think NoticeView is the best looking on the market so I extended it because I needed a notice view that could stick to the bottom for http://kanbana.com/ipad

I've added the feature as a property like this:

```
WBSuccessNoticeView *notice = [WBSuccessNoticeView successNoticeInView:rootView title:title];
notice.slidingMode = WBNoticeViewSlidingModeUp;
[notice show];
```

I've also fixed this issue https://github.com/tciuro/NoticeView/issues/28 

and an error when you tried to dissmis a nonSticky notice manually

I hope you'll merge this into your main repository and update the pod. I use Cocoa pods so I hope to use NoticeView as a pod in the future

Best Regards
Troels Richter
http://twitter.com/troelsrichter

troelsrichter committed:
noticeview with is calculated from the view bounds instead of frame to fix orientation with error

troelsrichter committed:
WBNoticeViewSlidingMode property introduced to support for sliding up from the bottom

troelsrichter committed:
there should never be a delay when notice is dismissed interactively. Fixed the case where you wanted to dismiss a non sticky notice interactively

troelsrichter committed:
ios5 special handling of notice height removed because it resultet in a higher view than else so that success and notice height differed on ios6

tciuro commented:
Hi Troels,

Thanks so much for taking the time to fix and improve NoticeView. I will take a look at it shortly and inspect the changes. I look forward merging it and make it available to everyone.

Regards,

-- Tito

adamfraser commented:
Would love this functionality as well.

troelsrichter committed:
error fixed: gradient view is redrawn to fit width if notice view is visible while the device is rotated
error fixed: sometimes the notice view was never dismissed if the app resigned active while a notice where shown

tciuro commented:
Thanks!
